android/fastlane/Fastfile

```diff
@@ -109,8 +109,8 @@ platform :android do
       track: 'beta',
       release_status: 'draft',
       skip_upload_apk: true,
-      package_name: 'org.pathcheck.covidsafepathsBte',
```

Comment:
This one i dont believe we can change at this point.

Reply:
👍 I will revert this and let's take a closer look at anything else we may want to revert.

Reply:
as long as we keep `package.json` in sync with the appId we're using in the android build files, we're ok. If we use an appId other than `org.pathcheck.covidsafepathsBte` it will create a new listing on the store, but now would be a better time to make the switch than later, if we do indeed want to migrate 🤔

Reply:
What do we think? I can go either way on this one.

I will change the appId back to Bte for now to get this in.

Reply:
if its fine to change, then i would prefer to use 'bt'

Reply:
let's go with `bt`  - I'll make sure Dick Lucas and Sam know 👍

Reply:
Awesome - I changed everything back to bt. Should I merge this or would you, @johnschoeman and @mattThousand, want to take one more look?